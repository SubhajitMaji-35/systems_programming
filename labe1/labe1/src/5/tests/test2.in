Ganga
Yamuna
Brahmaputra
Indus
Godavari
Krishna
Narmada
Tapi 
Mahanadi
Kaveri
