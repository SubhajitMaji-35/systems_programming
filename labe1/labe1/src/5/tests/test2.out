ana
amna
ramapura
nds
odvari
rihna
arada
ap 
ahnadi
avri
